src/ansys/aedt/core/workflows/circuit/import_schematic.py

```diff
@@ -38,26 +39,27 @@
 
 # Extension batch arguments
 extension_arguments = {"asc_file": ""}
-extension_description = "Import schematic to Circuit."
+extension_description = "Import schematic to Circuit"
 
 
 def frontend():  # pragma: no cover
-
     import tkinter
     from tkinter import filedialog
     from tkinter import ttk
 
     import PIL.Image
     import PIL.ImageTk
+    from ansys.aedt.core.workflows.misc import ExtensionTheme
 
     master = tkinter.Tk()
-
-    master.geometry("750x250")
-
+    master.geometry("650x150")
```

Comment:
@Samuelopez-ansys a small comment. I think that if you don't pass the size, tkinter will automatically fit the content in the window.